### PR TITLE
tiup: add close swap section and fix a command

### PR DESCRIPTION
**Pull request by kissmydb**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -1810,3 +1811,13 @@ cluster            YES(v0.4.6)  linux/amd64,darwin/amd64  Deploy a TiDB cluster
     ```bash
     tiup cluster exec tidb-test --sudo --command "yum -y install numactl"
     ```
+    
+### 如何关闭系统swap
+
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1816]: 这一节内容太单薄了，建议补充一句为什么要关闭系统 swap @kissmydb

File: how-to/deploy/orchestrated/tiup.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ category: how-to
     - 在 ARM 架构下，建议使用 CentOS 7.6 1810 版本 Linux 操作系统
 - TiKV 数据文件的文件系统推荐使用 EXT4 格式，也可以使用 CentOS 默认的 XFS 格式（参考[第 3 步](#第-3-步在-tikv-部署目标机器上添加数据盘-ext4-文件系统挂载参数)）
 - 机器之间内网互通（建议[关闭防火墙 `firewalld`](#如何关闭部署机器的防火墙)，或者开放 TiDB 集群的节点间所需端口）
+- 机器[关闭系统 swap](#如何关闭系统-swap)
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:51]:
```suggestion
- 所有部署主机[关闭系统 swap](#如何关闭系统-swap)
```

File: how-to/deploy/orchestrated/tiup.md [hunk 3]
```diff
@@ -1900,3 +1901,13 @@ cdc                             darwin/amd64,linux/amd64,linux/arm64
     ```bash
     tiup cluster exec tidb-test --sudo --command "yum -y install numactl"
     ```
+    
+### 如何关闭系统 swap
+
```
Comment (kissmydb) [inline how-to/deploy/orchestrated/tiup.md:1906]:
```suggestion

TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能，建议永久关闭系统 swap。并且不要使用 `swapoff -a` 关闭，重启会失效。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 4]
```diff
@@ -1900,3 +1901,14 @@ cdc                             darwin/amd64,linux/amd64,linux/arm64
...
+TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能，建议永久关闭系统 swap。并且不要使用 `swapoff -a` 关闭，重启会失效。
```
Comment (kissmydb) [inline how-to/deploy/orchestrated/tiup.md:1907]:
```suggestion
TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能，建议永久关闭系统 swap。并且不要使用 `swapoff -a` 关闭，重启会失效。

```

File: how-to/deploy/orchestrated/tiup.md [hunk 5]
```diff
@@ -1900,3 +1901,15 @@ cdc                             darwin/amd64,linux/amd64,linux/arm64
...
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1907]:
```suggestion
TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能。因此建议永久关闭系统 swap，并且不要使用 `swapoff -a` 方式关闭，否则重启机器后该操作会失效。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 6]
```diff
@@ -1900,3 +1901,15 @@ cdc                             darwin/amd64,linux/amd64,linux/arm64
     ```bash
     tiup cluster exec tidb-test --sudo --command "yum -y install numactl"
     ```
+    
+### 如何关闭系统 swap
+
+TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能。因此建议永久关闭系统 swap，并且不要使用 `swapoff -a` 方式关闭，否则重启机器后该操作会失效。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1907]:
```suggestion
TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能。因此建议永久关闭系统 swap，并且不要使用 `swapoff -a` 方式关闭，否则重启机器后该操作会失效。

建议执行以下命令关闭系统 swap：
```

File: how-to/deploy/orchestrated/tiup.md [hunk 7]
```diff
@@ -1345,7 +1346,7 @@ ID              Role          Host      Ports                            Status
 
 #### 查看 TiDB Dashboard 检查 TiDB 集群状态
 
-- 通过 `{pd-leader-ip}:2379/dashboard` 登录 TiDB Dashboard，登录用户和口令为 TiDB 数据库 root 用户和口令，如果你修改过数据库的 root 密码，则以修改后的密码为准，默认密码为空。
+- 通过 `{pd-ip}:2379/dashboard` 登录 TiDB Dashboard，登录用户和口令为 TiDB 数据库 root 用户和口令，如果你修改过数据库的 root 密码，则以修改后的密码为准，默认密码为空。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1349]: To make the commit from https://github.com/pingcap/docs-cn/pull/2919 in `master` branch

**Pull request merged**